assets/js/push.js

```diff
@@ -661,6 +670,11 @@ jQuery( window ).on( 'load', () => {
 					element.appendChild( removeLink );
 					element.classList = 'added-connection';
 
+					document.querySelector(
+						'.selected-connections-count'
+					).textContent =
+						'( ' + Object.keys( selectedConnections ).length + ' )';
```

Comment:
```suggestion
						'(' + Object.keys( selectedConnections ).length + ')';
```

Consistent presentation.

Reply:
@peterwilsoncc Yes, I agree with you. We should use `@wordpress/i18n` to make string translatable.

```js
import { sprintf, _n } from '@wordpress/i18n';

sprintf( _n( 'Selected connection (%d)', 'Selected connections (%d)', selectedConnectionsCount, 'text-domain' ), selectedConnectionsCount );
// Selected connection (1)'
// Selected connections (3)'
```

Reply:
@peterwilsoncc, I have used use @wordpress/i18n to make string translatable. 
CC : @ravinderk